fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBlockTags.java

```diff
@@ -0,0 +1,156 @@
+	 * and so, it is considered a special case and not given a storage block tag.
+	 */
+	public static final TagKey<Block> STORAGE_BLOCKS = register("storage_blocks");
+	public static final TagKey<Block> STORAGE_BLOCKS_BONE_MEAL = register("storage_blocks/bone_meal");
+	public static final TagKey<Block> STORAGE_BLOCKS_COAL = register("storage_blocks/coal");
+	public static final TagKey<Block> STORAGE_BLOCKS_COPPER = register("storage_blocks/copper");
+	public static final TagKey<Block> STORAGE_BLOCKS_DIAMOND = register("storage_blocks/diamond");
+	public static final TagKey<Block> STORAGE_BLOCKS_DRIED_KELP = register("storage_blocks/dried_kelp");
+	public static final TagKey<Block> STORAGE_BLOCKS_EMERALD = register("storage_blocks/emerald");
+	public static final TagKey<Block> STORAGE_BLOCKS_GOLD = register("storage_blocks/gold");
+	public static final TagKey<Block> STORAGE_BLOCKS_IRON = register("storage_blocks/iron");
+	public static final TagKey<Block> STORAGE_BLOCKS_LAPIS = register("storage_blocks/lapis");
+	public static final TagKey<Block> STORAGE_BLOCKS_NETHERITE = register("storage_blocks/netherite");
+	public static final TagKey<Block> STORAGE_BLOCKS_RAW_COPPER = register("storage_blocks/raw_copper");
+	public static final TagKey<Block> STORAGE_BLOCKS_RAW_GOLD = register("storage_blocks/raw_gold");
+	public static final TagKey<Block> STORAGE_BLOCKS_RAW_IRON = register("storage_blocks/raw_iron");
+	public static final TagKey<Block> STORAGE_BLOCKS_REDSTONE = register("storage_blocks/redstone");
+	public static final TagKey<Block> STORAGE_BLOCKS_SLIME = register("storage_blocks/slime");
+	public static final TagKey<Block> STORAGE_BLOCKS_WHEAT = register("storage_blocks/wheat");
+
+	// Misc
+	public static final TagKey<Block> PLAYER_WORKSTATIONS_CRAFTING_TABLES = register("player_workstations/crafting_tables");
+	public static final TagKey<Block> PLAYER_WORKSTATIONS_FURNACES = register("player_workstations/furnaces");
+	/**
+	 * Blocks should be included in this tag if their movement/relocation can cause serious issues such
+	 * as world corruption upon being moved or for balance reason where the block should not be able to be relocated.
+	 * Example: Chunk loaders or pipes where other mods that move blocks do not respect
+	 * {@link AbstractBlock.AbstractBlockState#getPistonBehavior}.
+	 */
+	public static final TagKey<Block> RELOCATION_NOT_SUPPORTED = register("relocation_not_supported");
+	/**
+	 * Tag that holds all head based blocks such as Skeleton Skull or Player Head. (Named skulls to match minecraft:skulls item tag)
+	 */
+	public static final TagKey<Block> SKULLS = register("skulls");
+	public static final TagKey<Block> ROPES = register("ropes");
+	public static final TagKey<Block> CHAINS = register("chains");
+
+	/**
+	 * Tag that holds all blocks that recipe viewers should not show to users.
+	 */
+	public static final TagKey<Block> HIDDEN_FROM_RECIPE_VIEWERS = register("hidden_from_recipe_viewers");
```

Comment:
Question: Why is this needed for blocks? Dont all recipe viewers only show the block items?

Reply:
Recipe viewers actually check for this tag for items, blocks, and fluids. A bit strange but there is a precedence for this so consolidating it under c should help out the recipe viewer mods
https://github.com/shedaniel/RoughlyEnoughItems/blob/fbdb76b16f971946302299b41e83abc3d86b81b0/runtime/src/main/java/me/shedaniel/rei/plugin/client/runtime/HideIngredientsFromTagsPlugin.java#L115-L119
![image](https://github.com/FabricMC/fabric/assets/40846040/9ca4d1bb-3433-4d0f-b480-0d2664aeb754)

Discord convo where this came up
https://discord.com/channels/313125603924639766/1134480199937957969/1187052300892655698

Reply:
Ok, sure at least there is a use case for it. Thanks for providing the details 👍